pyproject.toml

```diff
@@ -19,27 +19,24 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = ">=3.11, <4.0"
 edgegrid-python = "^1.2.1"
-python-json-logger = "^2.0.2"
-python-dotenv = ">=0.21.0,< 2"
-click = "^8.1.3"
-boto3 = "^1.26.3"
```

Comment:
these four are not explicitly used by this package so I'm eliminating them for now to avoid having too big of a surface area for cves.